src/compute/commitments.rs

```diff
@@ -139,6 +140,44 @@ pub fn compute_bls12_381_g1_commitments_with_generators(
     }
 }
 
+#[doc = include_str!("../../docs/commitments/compute_bn254_g1_commitments_with_generators.md")]
+///
+/// # Example - Pass generators to Commitment Computation
+///```no_run
+#[doc = include_str!("../../examples/pass_bn254_g1_generators_to_commitment.rs")]
+///```
+pub fn compute_bn254_g1_uncompressed_commitments_with_generators(
+    commitments: &mut [bn254_g1_affine],
+    data: &[Sequence],
+    generators: &[bn254_g1_affine],
+) {
+    init_backend();
```

Comment:
Should this be in here? Is it free if it's already been initialized?

Reply:
Great question. The `init_backend()` method call here just ensures the backend was in fact called. See [here](https://github.com/spaceandtimelabs/blitzar-rs/blob/e1ff8e0dcf79b87fd6b61969433361b71f0270c5/src/compute/backend.rs#L50-L70). If it has already been called, it simply returns without doing anything.

The other `compute_*_commitment` methods have the same call to `init_backend()`.